draft-ietf-avtcore-rtp-over-quic.md

```diff
@@ -931,6 +933,73 @@ As noted in {{multiplexing}}, all RoQ streams and RoQ datagrams begin with a flo
 
+## RTCP Considerations
+
+RTCP was originally defined to be used with UDP, which implies (1) that
+transmission timing is largely under the control of the application (limited
+buffering inside the stack) and (2) that the overhead, *avg_rtcp_size*, used to
+compute the RTCP transmission interval could be deterministically computed by
+adding the IP asnd UDP headers. Both change when carrying RTCP over QUIC and
+they change in different ways when using QUIC streams vs. QUIC datagrams.
+
+### RTCP over QUIC datagrams {#rtcp-over-datagrams}
+
+When sending RTCP packets in QUIC datagrams this implies that a packet may not
+be immediately transmitted as it is subject to queuing and multiplexing with RTP
+packets and subject to congestion control. This means that a sending timestamp
+added to an RTCP packet, e.g., in an SR packet, may differ in unforeseeable ways
+from the actual time when the packet gets sent into the network while these are
+usually fairly close to each other for RTP-over-UDP. Effectively, we have a
+application sending timestamp *t_a* and the network transmission timestamp
+*t_n*. Applications just have to be aware that RTCP does not measure the network
+level RTT but rather the application layer RTT.
+
+Moreover, the true overhead per RTCP packet cannot easily be determined: this is
+because, in addition to adding the IP and UDP headers, the QUIC (short) header
+and the QUIC datagram frame header are to be considered but their sizes vary and
+it is unknown which other frames may be sent along in the same UDP packet. Any
+lower bound that can be determined could be affected by the version of QUIC
+being used.
+
+It is thus suggested that applications do _not_ attempt to include lower layer
+headers but only work with the net data rate of the codec including the RTP
+headers and of the RTCP packets, both excluding QUIC, UDP, and IP header
+overhead.
+
+> **Editor’s Note:** For compatibility with the computation of RTP-over-UDP
+> implementations, one could consider adding just the UDP and IP headers.
+
+### RTCP over QUIC streams
+
+The above considerations from {{rtcp-over-datagrams}} get even more complex when
+transmitting RTCP reliably over QUIC streams: it is unknown if (and how many)
+retransmissions occurred.
+
+For RTT computations, again, this means that the application must consider that
+it observes the application layer RTT including retransmissions, where
+retransmissions also contribute to the observed jitter.
+
+For overhead computation, retransmissions are not explicitly considered nor is
+the multiplexing with other streams.
+
+To keep the complexity under control, it is again recommended to _not_ attempt
+to include the unknown lower layer overhead and only use the data rates and
+packet sizes visible at the application (=RTP/RTCP) layer.
+
+> **Editor’s note:** previous note also applies here but, due to
+> retransmissions, overhead is even less predictable.
```

Comment:
```suggestion

```
This is just removing the Editor's note, which refers to a previous Editor's note that no longer exists.